src/zoid/checkout/component.jsx

```diff
@@ -154,6 +154,12 @@ export function getCheckoutComponent() : ZoidComponent<CheckoutPropsType> {
                     queryParam: true,
                     default:    () => FUNDING.PAYPAL
                 },
+
+                standaloneFundingSource: {
```

Comment:
We already have `fundingSource` param which signifies a standalone button, we should rely on that.

Reply:
This is the flag to identify if we override some rules because of enablefunding param 
https://github.com/paypal/paypal-smart-payment-buttons/blob/897f0a7da2c59cf119bbc08c777fd77760cda956/src/button/props.js#L219

Reply:
I don't think we need an additional prop for this, right? This can be derived any time it's needed.

Reply:
Agreed to remove this right?